src/main/java/com/codeborne/selenide/impl/AttachmentConsolePrinter.java

```diff
@@ -0,0 +1,22 @@
+package com.codeborne.selenide.impl;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+
+public class AttachmentConsolePrinter {
```

Comment:
@vtintillier Can you please make this class non-public (just remove the "public" keyword)?